src/PricingModules/StandardERC20PricingModule.sol

```diff
@@ -77,6 +77,36 @@ contract StandardERC20PricingModule is PricingModule, IStandardERC20PricingModul
         IMainRegistry(mainRegistry).addAsset(asset);
     }
 
+    /**
+     * @notice Sets a new oracle sequence in the case one of the oracles is decomissioned.
+     * @param asset The contract address of the asset
+     * @param newOracles An array of addresses of oracle contracts, to price the asset in USD
+     * @param decommissionedOracle The contract address of the decommissioned oracle
+     */
+    function setOracles(address asset, address[] calldata newOracles, address decommissionedOracle)
```

Comment:
should we put this in the abstract?

Reply:
We can't (assetInformation is a different struct for each PricingModule) and we shouldn't (PricingModules of derived assets, like aTokens don't need setters, they should fetch the oracles from the PricingModule of the underlying asset)